subprojects/studio-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,22 @@
+<idea-plugin>
```

Comment:
Does this file need to be checked in? It seems like the build file also adds the plugin dependencies.

Reply:
This file is required since this is a plugin descriptor and is packed into the jar and later read by IntelliJ. Dependencies are unfortunately duplicated. Added a comment that dependency should be added to intellij extension

Reply:
OK.